api/v1/ptpoperatorconfig_types.go

```diff
@@ -58,6 +61,15 @@ type PtpOperatorConfigList struct {
 	Items           []PtpOperatorConfig `json:"items"`
 }
 
+// PtpEventConfig defines the desired state of event framework
+type PtpEventConfig struct {
+	// +kubebuilder:default=false
+	// EnableEventPublisher will deploy event proxy as a sidecar
+	EnableEventPublisher bool `json:"enableEventPublisher,omitempty"`
+	// transportHost address for event messages.Example amqp://<service-name>.<namespace>.svc.cluster.local
+	TransportHost string `json:"transportHost,omitempty"`
```

Comment:
should you have a port here also?

Reply:
The sidecar rest api port is not that dynamic. The port is an internal sidecar rest  API that is only accessible by the ptp-operator-plugin via localhost only. it is not used by anything else.

Reply:
amqp transportHost doesn't have a port, we just use service name like `amqp://<service-name>.<namespace>.svc.cluster.local`